src/data/full-quiz.js

```diff
@@ -1,4 +1,13 @@
 const fullQuiz = [
+  {
+    Question: "Which of the following is correct for arrays?",
+    Answer: "Size of an array can not be changed once intialized",
+    Distractor1: "Insertion and deletion of an element is efficient",

```

Comment:
I'd like to modify Distractor1's answer. 
Proposed fix: Insertion and deletion of an element has a time complexity of O(1).